javaparser-core-testing/src/test/java/com/github/javaparser/printer/PrettyPrinterTest.java

```diff
@@ -47,19 +49,23 @@
 import com.github.javaparser.ast.expr.VariableDeclarationExpr;
 import com.github.javaparser.ast.stmt.Statement;
 import com.github.javaparser.ast.type.PrimitiveType;
+import com.github.javaparser.ast.visitor.VoidVisitor;
+import com.github.javaparser.printer.configuration.ConfigurablePrinter;
 import com.github.javaparser.printer.configuration.Indentation;
 import com.github.javaparser.printer.configuration.Indentation.IndentType;
+import com.github.javaparser.printer.configuration.PrinterConfiguration;
+import com.github.javaparser.printer.configuration.PrinterConfiguration.ConfigOption;
 
 class PrettyPrinterTest {
```

Comment:
Forgot to rename the class?

Reply:
No, this is the actual code. I'm just adapting it to use a PrinterConfiguration (new class) internally.